source/GameInterface/Services/PartyComponents/Patches/Lifetime/BanditPartyComponentLifetimePatches.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Policies;
+using GameInterface.Services.PartyComponents.Messages;
+using HarmonyLib;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using TaleWorlds.CampaignSystem.Party;
+using TaleWorlds.CampaignSystem.Party.PartyComponents;
+
+namespace GameInterface.Services.PartyComponents.Patches.Lifetime;
+
+
+/// <summary>
+/// Harmony patches for the lifetime of a <see cref="BanditPartyComponent"/> object
+/// </summary>
+[HarmonyPatch]
+internal class BanditPartyComponentLifetimePatches
+{
+    private static readonly ILogger Logger = LogManager.GetLogger<BanditPartyComponentLifetimePatches>();
+
+    private static IEnumerable<MethodBase> TargetMethods() => AccessTools.GetDeclaredConstructors(typeof(BanditPartyComponent));
+
+    private static bool Prefix(BanditPartyComponent __instance)
+    {
+        // Skip if we called it
+        if (CallOriginalPolicy.IsOriginalAllowed()) return true;
+
+        if (ModInformation.IsClient)
+        {
+            Logger.Error("Client created unmanaged {name}\n"
+                + "Callstack: {callstack}", typeof(MobileParty), Environment.StackTrace);
```

Comment:
Shouldn't it be "typeof(BanditPartyComponent)" or is it part of the MobileParty?

Reply:
That should be the patched type, updated